src/link/Elf.zig

```diff
@@ -4128,7 +4128,7 @@ pub fn getTarget(self: Elf) std.Target {
 fn requiresThunks(self: Elf) bool {
     return switch (self.getTarget().cpu.arch) {
         .aarch64 => true,
-        .x86_64, .riscv64 => false,
+        .x86_64, .riscv64, .loongarch64 => false,
```

Comment:
Worth checking if all the added linker code is actually 64-bit only; I suspect not?